source/wp-content/themes/wporg-developer-2023/templates/single-blocks-handbook.html

```diff
@@ -0,0 +1,85 @@
+<!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
+<main class="wp-block-group alignfull" style="padding-left:var(--wp--preset--spacing--edge-space);padding-right:var(--wp--preset--spacing--edge-space)">
+
+	<!-- wp:group {"layout":{"type":"constrained","justifyContent":"left"},"align":"wide"} -->
+	<div class="wp-block-group alignwide">
+
+		<!-- wp:group {"tagName":"article"} -->
+		<article class="wp-block-group">
+			<!-- wp:post-title {"level":1,"style":{"spacing":{"margin":{"top":"var:preset|spacing|edge-space","bottom":"var:preset|spacing|40"}}}} /-->
+
+			<!-- wp:group {"tagName":"aside","className":"sidebar-container"} -->
+			<aside class="wp-block-group sidebar-container">
+				<!-- wp:search {"label":"Search","showLabel":false,"widthUnit":"px","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","borderColor":"light-grey-1","placeholder":"Search the documentation"} /-->
```

Comment:
This is dropping into the code search results. We'll need to update that at some point.

Reply:
This is all done in a `searchform.php` template in the old theme… we can probably do something like [the global header search](https://github.com/WordPress/wporg-mu-plugins/blob/16ee1d69b809ac6d4e1b6963e0192f2dd4802af9/mu-plugins/blocks/global-header-footer/blocks.php#L999-L1016).

At least it looks like the search is mostly works as long as we set the form URL correctly:

![Screen Shot 2023-01-24 at 16 13 26](https://user-images.githubusercontent.com/541093/214417462-1c682fb3-3a18-4177-adc0-85e59f28a612.png)
http://localhost:8888/block-editor/?s=test